WebHandler/scrappers/nclt.py

```diff
@@ -0,0 +1,51 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.support.ui import Select
+from azure.storage.blob import BlobServiceClient
```

Comment:
write a utility method for storing files in azure and re-use that function everywhere.